### PR TITLE
Feature (zeb-982): tests batch actions

### PR DESCRIPTION
**Pull request by danymill**
image

image

### CODE REVIEW

File: client/app/containers/test-details/test-details.controller.js [hunk 1]
```diff
@@ -319,6 +327,45 @@ const testDetailsController = function testDetailsController(
         }
     }
 
+    function bulkChangeStatus(event, status) {
+        if (vm.bulkChangeInProgress) { return; }
+        const btn = event.currentTarget;
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.controller.js:332]: rewrite or comment what do you do that

File: client/app/containers/test-details/test-details.controller.js [hunk 2]
```diff
@@ -87,6 +87,9 @@ const testDetailsController = function testDetailsController(
         zafiraWebsocket: null,
         testId: null,
         configSnapshot: null,
+        selectedTests: 0,
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.controller.js:90]: `selectedTests` the name looks like you have an array with tests, not a count of tasts

**Review by dzmitrymishyn — CHANGES_REQUESTED** [review]

File: client/app/_services/test.service.js [hunk 3]
```diff
@@ -94,5 +94,20 @@
             return $httpMock.get(API_URL + '/api/tests/jira/connect').then(UtilService.handleSuccess, UtilService.handleError('Unable to get connection to Jira'));
         }
 
+        /**
+         *
+         * @param {Number[]} data - Array of IDs
+         * @param {String} status
+         * @returns {PromiseLike<any> | Promise<any>}
+         */
+        function updateTestsStatus(data = [], status = '') {
```
[the remark this answers is not in the document]
Comment (danymill) [reply client/app/_services/test.service.js]: fixed

File: client/app/containers/test-details/test-details.controller.js (hunk 1 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.controller.js]: fixed

File: client/app/containers/test-details/test-details.controller.js (hunk 2 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.controller.js]: fixed

**Review by dzmitrymishyn — APPROVED** [review]

**Pull request merged**